### DISCUSSION TITLE
Difference between find and search

### DISCUSSION THREAD

**Discussion by bitfoo1** [gokcehan/lf#1545] [Q&A] [answered]
Hi,
I found that there are two different "searches" (search "/" and find "f").
If not changing `findlen (default 1)` the `find` command exits with `find: pattern not found: c` if there is no file/directory beginning with this letter. 

Actually I do not understand what this command is about hand what benefit it can give. 

Disclaimer: The benefit is not clear to me! Not that I say it has no. ;-)

Thank you for your help.

**Comment by joelim-work** [comment, marked as answer]
Although `find` is less flexible than `search`, it requires less keystrokes and is therefore faster.

For example to find the next file beginning with 'c', you can type `fc` instead of `/c<enter>`. Technically `/c<enter>` will match any 'c', not just at the beginning of the filename, so you would need to enable `globsearch` and type `/c*<enter>`.

**Comment by bitfoo1** [comment]
Thank you @joelim-work, that meets my expectations!
